api-reference/beta/resources/applicationsecurityprofile.md

```diff
@@ -0,0 +1,91 @@
+# applicationSecurityProfile resource type
+
+ > **Important:** APIs under the /beta version in Microsoft Graph are in preview and are subject to change. Use of these APIs in production applications is not supported.
+
+Defines the relevant security properties of applications to provide additional context for alerts and targets for actions.
+
+Security profiles are populated by integrated solutions at their discretion. For example: 
+
+- A solution will populate hostSecurityProfile for any desktop, laptop, server, firewall, or other endpoint that their service is responsible for monitoring, alerting, or taking threat remediation actions against. 
+- A provider will populate **ipSecurityProfile** for a collected set of IP addresses that their service is responsible for isolating, monitoring, or configuring.
+
+## Methods
+
+| Method   | Return Type |Description|
+|:---------------|:--------|:----------|
+|[Get applicationSecurityProfile](../api/applicationsecurityprofile_get.md) | [applicationSecurityProfile](applicationsecurityprofile.md) |Read properties and relationships of applicationSecurityProfile object.|

```

Comment:
The link text should just say "Get security profile" and so on. Also add "an" before applicationSecurityProfile.